index.js

```diff
@@ -117,15 +117,21 @@ module.exports = class Transaction {
    * @return {Buffer}
    */
   hash (signature) {
+    const cpyRaw = this.raw.slice(0)
```

Comment:
Please don't abbreviate, `rawCopy` should be more readable. Why is this variable needed?